### ISSUE TITLE
Nested keys and commented out keys generation bug

### ISSUE THREAD

**Issue by mme-private** [ngneat/transloco-keys-manager#10]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

For module:
```ts
@NgModule({
	imports: [
		...
		TranslocoModule,
	],
	declarations: [
		...
	],
	providers: [{ provide: TRANSLOCO_SCOPE, useValue: 'property' }],
})
export class PropertyModule { }
```

And some component template in this module:
```html
<ng-container *transloco="let t; read: 'property'">
			<ng-container *transloco="let t;">
				<h3 class="modal-title">{{ t('some.global.title1') }}</h3>
			</ng-container>

			<!-- <ng-container *transloco="let t;">
				<h3 class="modal-title">{{ t('some.global.key1') }}</h3>
			</ng-container> -->

                        <!-- <h3 class="modal-title">{{ t('propname') }}</h3> -->
```
There are extracted keys:
assets/i18n/en.json:
```js
{
  "some.global.title1": "some.global.title1"
}
```
assets/i18n/property/en.json:
```js
{
  "propname": "property.propname",
  "some.global.key1": "property.some.global.key1",
  "some.global.title1": "property.some.global.title1"
}
```

## Expected behavior
There should be only one key generated in assets/i18n/en.json file 'some.global.title1' key.

So there is two problems:
1) nested *transloco extract two keys, when there should be only one
2) for some reason commented block keys is also generated, but only for outer *transloco directive

## Environment

<pre><code>
Angular version: 8.2.14
 
For Tooling issues:
- Node version: v12.13.1  
- Platform:  Windows 7
</code></pre>

**Comment by NetanelBasal** [comment]
@mme-private we don't support nested directives with the same name. You can change one of the `let t` variables, and it should work.

Regarding the comments, @shaharkazaz will fix it soon.

**Comment by shaharkazaz** [comment]
@mme-private comments are fixed in v1.2.1
